pythonbits/bb.py

```diff
@@ -365,45 +424,67 @@ def _render_container(self):
             return 'AVI'
         elif general['format'] == 'MPEG-4':
             return 'MP4'
+        elif general['format'] == 'BDAV':
+            return 'm2ts'
         else:
-            raise Exception("Unknown or unsupported container", general.format)
+            raise Exception("Unknown or unsupported container: %r" % (general.format,))
 
     def _render_video_codec(self):
         video_track = self['tracks']['video']
         # norm_bitrate = (float(bit_rate) /
         #     (video_track.width*video_track.height))
-        if video_track['codec'] in ('V_MPEG4/ISO/AVC', 'AVC'):
-            if ('writing_library' in video_track and
-                    'x264' in video_track['writing_library']):
+        if (video_track['codec_id'] in ('V_MPEG4/ISO/AVC', 'AVC', 'avc1') or
+            video_track['format'] == 'AVC'):
+            if 'x264' in video_track.get('writing_library', ''):
                 return 'x264'
             else:
                 return 'H.264'
-        elif video_track['codec'] == 'XVID':
+        elif video_track['codec_id'] in ('V_MPEGH/ISO/HEVC', 'HEVC'):
+            if 'x265' in video_track.get('writing_library', ''):
+                return 'x265'
+            else:
+                return 'H.265'
```

Comment:
Make style consistent

Reply:
Looks perfectly readable to me. I don't see any inconsistencies.

Reply:
You're right, I'm stupid. I misread the diff. It's good.